code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -416,3 +420,55 @@
 	var/datum/role/wizard/summon_magic/S = R
 	if(istype(S))
 		S.summons_received = randomizemagic
+
+/mob/living/carbon/human/proc/equip_potions(var/datum/role/R)
+	for(var/i=0, i<3, i++)
```

Comment:
maybe create a `get_random_potion()` proc. It could be reused on map spawners for example

Reply:
that's a good idea, should I just make it a global proc?

Reply:
Yes. Maybe in unsorted.dm. Maybe there's a better place